app/Routes.hs

```diff
@@ -48,8 +47,9 @@ strictRoutes aboutContents privacyContents = [
     ("post", retrievePost),
     ("post-progress", postResponse),
     ("draw", drawResponse),
-    ("about", aboutResponse aboutContents),
-    ("privacy", privacyResponse privacyContents),
+    ("about", aboutResponse),
+    ("PRIVACY.md", privacyResponse),  -- check for this route in strictMatchDir and redirect to /privacy?
```

Comment:
Okay, I looked into this some more. I don't actually think it's worth generating the privacy policy as an HTML file, and instead I'd like to modify the link to refer directly to https://github.com/Courseography/courseography/blob/master/PRIVACY.md, and similarly for the LICENSE. Please make those changes in the README.md file, and then *remove* all privacy-policy-related routes/functions/files in this PR.